deid/tests/test_replace_identifiers.py

```diff
@@ -118,16 +118,12 @@ def test_replace_with_constant(self):
         ]
         recipe = create_recipe(actions)
 
```

Comment:
Explanation of the change below:

After implementing the change for sequence processing, this unit test was failing with an index out of bounds exception.  This exception was occurring because the DicomParser object was applying the stock deid.dicom recipe which was causing the Accession Number field to be removed.  I did not dig into why the sequence change affected this unittest in this manner, but since it was only impacting the check for getting the "before" values and all other unit tests passed, I didn't feel that it warranted further investigation. Instead I just updated this to red the file and directly retrieve the "before" values